<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToasterWindow</class>
 <widget class="QMainWindow" name="ToasterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>694</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>926</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>926</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Toaster</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>900</width>
         <height>580</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>900</width>
         <height>580</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="MainFrame" name="mainFrame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>900</width>
            <height>580</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>900</width>
            <height>580</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">MainFrame#mainFrame {background-image: url(:/Images/resources/images/KPA_MainWindow_Dark_Skin.png);}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionUploadKIPRFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>&amp;Connection</string>
    </property>
    <addaction name="actionOpenMIDIPorts"/>
    <addaction name="actionClose_MIDI_Ports"/>
    <addaction name="separator"/>
    <addaction name="actionConnectToKPA"/>
    <addaction name="actionDisconnectFromKPA"/>
    <addaction name="separator"/>
    <addaction name="actionRequestValues"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settin&amp;gs</string>
    </property>
    <addaction name="actionShowStatusbar"/>
    <addaction name="separator"/>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Deb&amp;ug</string>
    </property>
    <addaction name="actionCreateStringValuesTable"/>
    <addaction name="actionSendSySex"/>
    <addaction name="actionDebugSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnection"/>
   <addaction name="menuSettings"/>
   <addaction name="menuDebug"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionRequestValues">
   <property name="text">
    <string>&amp;Request Values</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>&amp;Configure Toaster...</string>
   </property>
   <property name="toolTip">
    <string>Configure Toaster</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCmd">
   <property name="text">
    <string>Cmd</string>
   </property>
  </action>
  <action name="actionOpenMIDIPorts">
   <property name="text">
    <string>&amp;Open MIDI Ports</string>
   </property>
  </action>
  <action name="actionConnectToKPA">
   <property name="text">
    <string>Connect To &amp;KPA</string>
   </property>
  </action>
  <action name="actionDisconnectFromKPA">
   <property name="text">
    <string>&amp;Disconnect From KPA</string>
   </property>
  </action>
  <action name="actionShowStatusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Statusbar</string>
   </property>
  </action>
  <action name="actionUploadKIPRFile">
   <property name="text">
    <string>&amp;Upload KIPR File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionClose_MIDI_Ports">
   <property name="text">
    <string>&amp;Close MIDI Ports</string>
   </property>
  </action>
  <action name="actionCreateStringValuesTable">
   <property name="text">
    <string>&amp;Create String Values Table...</string>
   </property>
   <property name="toolTip">
    <string>Create String Values Table</string>
   </property>
  </action>
  <action name="actionSendSySex">
   <property name="text">
    <string>&amp;Send SySex...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionDebugSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainFrame</class>
   <extends>QFrame</extends>
   <header>MainFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionShowStatusbar</sender>
   <signal>toggled(bool)</signal>
   <receiver>statusBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>678</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
